packages/backend-test-utils/src/next/implementations/mockIdentityService.ts

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2023 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import {
+  coreServices,
+  createServiceFactory,
+  IdentityService,
+} from '@backstage/backend-plugin-api';
+import {
+  IdentityApiGetIdentityRequest,
+  BackstageIdentityResponse,
+} from '@backstage/plugin-auth-node';
+
+class MockIdentityServiceImpl implements IdentityService {
+  getIdentity(
+    _options: IdentityApiGetIdentityRequest,
+  ): Promise<BackstageIdentityResponse | undefined> {
+    return Promise.resolve({
+      token: 'mock-token',
+      identity: {
+        type: 'user',
+        userEntityRef: 'user:default/mockUser',
```

Comment:
not fond of this having uppercase chars in it, which is invalid in refs :) could use mock-user instead